lib/src/command/outdated.dart

```diff
@@ -65,11 +64,9 @@ class OutdatedCommand extends PubCommand {
     entrypoint.assertUpToDate();
 
     final includeDevDependencies = argResults['dev-dependencies'];
-
     final upgradePubspec = includeDevDependencies
-        ? entrypoint.root.pubspec
+        ? _stripOverrides(entrypoint.root.pubspec)
         : _stripDevDependencies(entrypoint.root.pubspec);
```

Comment:
Need to rename this `stripOverridesAndDevDeps` or we do:
`_stripDevDependencies(_stripOverrides(entrypoint.root.pubspec))`.

And fix the todo in `_stripDevDependencies` by preserving `dependency_overrides`.